editor/sidebar/post-author/index.js

```diff
@@ -73,15 +74,23 @@ class PostAuthor extends Component {
 	}
 }
 
-export default connect(
-	( state ) => {
+export default flowRight( [
+	connect(
+		( state ) => {
+			return {
+				postAuthor: getEditedPostAttribute( state, 'author' ),
+			};
+		},
+		{
+			onUpdateAuthor( author ) {
+				return editPost( { author } );
+			},
+		},
+	),
+	withAPIData( () => {
 		return {
-			postAuthor: getEditedPostAttribute( state, 'author' ),
+			users: '/wp/v2/users?context=edit&per_page=100',
 		};
-	},
-	{
-		onUpdateAuthor( author ) {
-			return editPost( { author } );
-		},
-	},
-)( withInstanceId( PostAuthor ) );
+	} ),
```

Comment:
Do you think we should enforce a convention here, something like this;

```js
const connectComponent = connect( ... );
const fetchAPIData = withApiData( ... );

export default flowRight( [
  connectComponent,
  fetchAPIData,
] )( Component );
```

Reply:
Out of curiosity, what's your reason for wanting a convention here? Are you finding this could be difficult to interpret?

Reply:
My primary reason is to avoid indentation changes when moving from one HoC to composed HOCs. But I also find it more readable.